bedrock/firefox/templates/firefox/dashboard-base.html

```diff
@@ -7,7 +7,7 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox-dashboard' %}
```

Comment:
So `{% block extrahead %}` is basically a general purpose container for anything you might want to put in the `<head>` - CSS, JS, `<meta>` tags, etc. If you place CSS bundles in it, they will be shown to *all* browsers, whereas CSS bundles placed in `site_css` and `page_css` will be hidden from IE 7 and below. In short, it's a way to force CSS on IE 7 and lower.